### DISCUSSION TITLE
🎹 Sharing Samples!

### DISCUSSION THREAD

**Discussion by hankydotorg** [FundamentalFrequency/.github#119] [Show and tell]
In the spirit of [briwil's post generously sharing his drum samples & YAML files](https://github.com/orgs/FundamentalFrequency/discussions/51) we thought we would start a new discussion forum for anyone and everyone to share their sample libraries -- especially for people who don't yet have their own libraries to start.

**Comment by raychmond** [comment]
This is a [copy](https://1drv.ms/f/s!Ak3OXcjNGLBXicov2xFdzFTFDQswDw?e=4VwFiY) of original briwil's drum samples & YAML files. Thanks @hankydotorg for initiating this and creating the new post.

**Reply by hankydotorg** [reply]
Thanks @raychmond - I've linked my samples below! Hope that it persuades you to bring your LMN-3 out to play again!

**Comment by hankydotorg** [comment]
I've put together [my drum_kits and samples to download here](https://ln5.sync.com/dl/6605a5750/um6fytkb-kzffc79c-cex85e8y-9y34kprr). More samples than drum kits.  

Please note I am not a programmer, and my attempts at the YAML may be somewhat lacklustre (still worked on my LMN-3)!

I still have some more samples I'm gathering and will update with some more soon!

**Reply by raychmond** [reply]
Incredible! Thanks a lot for sharing so much good stuff @hankydotorg I will definitely try these new samples.
